.github/workflows/main.yml

```diff
@@ -0,0 +1,52 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. Triggers the workflow on push or pull request
+# events but only for the master branch
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
```

Comment:
Yes, this sounds like the right idea. Am I getting this right that this action is triggered on pushes and PR merges (?) to master? 

If so, I could adjust the workflow for contributions by introducing a /dev branch, to which hotfixes will be pushed and then I regularly merge to master, which then (if I understand correctly) triggers the action. In this way, we avoid spamming _too_ many releases.

Reply:
The thing is that I have _no_ experience with github actions. The thing that's not entirely clear to me here is whether the _request_ is the trigger or the merge of the request (the latter being desireable).

Reply:
I updated it to only build on push, this way were not creating to many releases linke you said

Reply:
Great! Thanks (y)